src/components/entityset/EntitySetList.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+
+import { EntitySetSummary } from './EntitySet';
+import styles from '../../containers/catalog/catalog.module.css';
+
+export class EntitySetList extends React.Component {
+  static propTypes = {
+    entitySets: PropTypes.array.isRequired
```

Comment:
+1. I'm not thrilled with how I'm doing `propTypes` throughout any of this code. I think using the `EntitySet` object you're creating in `loom-data-js` is the right next step. Flow types seems like another good option, but they weren't actually being validated in our build.